pkg/server/file.go

```diff
@@ -121,6 +121,11 @@ func (t *TranslateServiceServer) UploadTranslationFile(
 		return nil, status.Errorf(codes.InvalidArgument, err.Error())
 	}
 
+	allMessages, err := t.repo.LoadMessages(ctx, params.serviceID, common.LoadMessagesOpts{})
```

Comment:
The flow of uploading translation file can be simplified a lot. Why do we have two loads, two saves, two populated messages etc?

Reply:
forgot to remove the previous populate messages